homeassistant/components/olarm/strings.json

```
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "device": {
+        "title": "Select Device",
```

This should be sentence-cased:

```suggestion
        "title": "Select device",
```